docs/get-started/what-is.md

```diff
@@ -4,29 +4,29 @@
 :scale: 0 %
 ```
 
-- It is like JSON, but for intensive computation.
-- It is like `numpy.ndarray`, but for unstructured data. 
-- It is like `pandas.DataFrame`, but for nested and mixed media data with embeddings.
-- It is like Protobuf, but for data scientists and deep learning engineers. 
+- It's like JSON, but for intensive computation.
+- It's like `numpy.ndarray`, but for unstructured data. 
+- It's like `pandas.DataFrame`, but for nested and mixed media data with embeddings.
+- It's like Protobuf, but for data scientists and deep learning engineers. 
 
-If you are a **data scientist** who works with image, text, video, audio data in Python all day, you should use DocArray: it can greatly accelerate the work on representing, embedding, matching, visualizing, evaluating, sharing data; while stay close with your favorite toolkits, e.g. Torch, Tensorflow, ONNX, PaddlePaddle, JupyterLab, Google Colab.
+If you are a **data scientist** who works with image, text, video, audio data in Python all day, you should use DocArray: it can greatly accelerate the work on representing, embedding, matching, visualizing, evaluating, sharing data; while staying close to your favorite toolkits, e.g. Torch, TensorFlow, ONNX, PaddlePaddle, JupyterLab, Google Colab.
 
-If you are a **deep learning engineer** who works on scalable deep learning service, you should use DocArray: it can be the basic building block of your system. Its portable data structure can be wired in Protobuf, compressed bytes, JSON; allowing your engineer friends to happily integrate it into the production system.
+If you are a **deep learning engineer** who works on scalable deep-learning services, you should use DocArray: it can be the basic building block of your system. Its portable data structure can be wired in Protobuf, compressed bytes, JSON; allowing your engineer friends to happily integrate it into the production system.
```

Comment:
```suggestion
If you are a **deep learning engineer** who works on scalable deep learning services, you should use DocArray: it can be the basic building block of your system. Its portable data structure can be wired in Protobuf, compressed bytes, JSON; allowing your engineer friends to happily integrate it into the production system.
```

isn't it okay to say deep learning ? I do have the feeling that everybody write it like this